lang/java/reef-runtime-local/src/main/java/org/apache/reef/runtime/local/client/parameters/RootFolder.java

```diff
@@ -26,7 +26,7 @@
  */
 @NamedParameter(default_value = RootFolder.DEFAULT_VALUE, doc = "The folder where logs etc. shall be stored.")
 public final class RootFolder implements Name<String> {
-  public static final String DEFAULT_VALUE = "REEF_LOCAL_RUNTIME";
+  public static final String DEFAULT_VALUE = "target/REEF_LOCAL_RUNTIME";
```

Comment:
Can we instead bind the folder in the `LocalTestEnvironment`? That way, the change is much more local and don't change behavior for people who rely on the current values.